### PR TITLE
feat: makes the `created/updated` indicators on ORM models optional.

### PR DESCRIPTION
**Pull request by cofin**
I'd like to be able to optionally include the 2 audit columns on base models.  

This enhancements moves the created/updated into a mixin and updates the test cases accordingly.  

If you are OK with something like this, I'll make updates to the documentation next.

### CODE REVIEW

File: tests/unit/test_worker.py [hunk 1]
```diff
@@ -20,5 +20,5 @@ def test_worker_decoder_handles_pydantic_models(authors: list[Author]) -> None:
     encoded = worker.encoder.encode(pydantic_model)
     assert (
         encoded
-        == b'{"id":"97108ac1-ffcb-411d-8b1e-d9183399f63b","created":"0001-01-01T00:00:00","updated":"0001-01-01T00:00:00","name":"Agatha Christie","dob":"1890-09-15"}'
+        == b'{"created":"0001-01-01T00:00:00","updated":"0001-01-01T00:00:00","id":"97108ac1-ffcb-411d-8b1e-d9183399f63b","name":"Agatha Christie","dob":"1890-09-15"}'
```
Comment (cofin) [inline tests/unit/test_worker.py:23]: I tried settings the id column position to 1 (`id._column_position = 1`), but the mixin still listed the audit columns first.

Comment (peterschutt) [reply tests/unit/test_worker.py:23]: Maybe something like this: https://stackoverflow.com/a/4013184/6560549

**Comment by peterschutt** [comment]
Hey mate, this will be no problem, there's prob a bit of breaking stuff coming through with #213 and #257, so might bundle those and this together for a 0.29 release.

Don't stress too much on the docs, they ground-up work once I get the api settled.

maybe have 2 bases in there `orm.Base` and `orm.AuditBase` that includes the extra columns so can choose one or the other.

**Comment by peterschutt** [comment]
closed in favor or #264

**Pull request closed without merging**